custom_components/sensor/hue_sensors.py

```diff
@@ -17,8 +17,9 @@
 _LOGGER = logging.getLogger(__name__)
 PHUE_CONFIG_FILE = 'phue.conf'
 SCAN_INTERVAL = timedelta(seconds=1)
-
-
+#... I preffer this naming, but happy to match yours
```

Comment:
Thanks for the contribution :-) 
Could you run the style checking with tox as per https://home-assistant.io/developers/development_testing/ 
Certainly tox will complain about the formatting of these comments.
Once tox is happy then can merge.
Cheers!

Reply:
Actually don't worry, tox only required for submission to HA which I'm handling in a different repo